Sources/TokamakStaticHTML/PublishSupport/FoundationTokamakFactory+Views.swift

```diff
@@ -0,0 +1,124 @@
+#if canImport(Publish)
+import Foundation
+import Publish
+
+extension FoundationTokamakFactory {
+  struct ItemList: View {
+    let items: [Item<Site>]
+    let site: Site
+
+    var body: some View {
+      VStack(alignment: .leading) {
+        ForEach(items) { item in
+          VStack(alignment: .leading) {
+            Link(destination: URL(string: item.path.absoluteString)!) {
+              Text(item.title)
+                .font(.headline)
+                .underline()
+            }
+            TagList(item: item, site: site)
+              .padding(.vertical)
+            Text(item.description)
+              .font(.caption)
+          }
+          .padding(20)
+          .background(Color(0xEEEEEE))
+          .cornerRadius(20)
+        }
+      }
+    }
+  }
+
+  struct TagList: View {
+    let item: Item<Site>
+    let site: Site
+
+    var body: some View {
+      HStack(spacing: 5) {
+        ForEach(item.tags) { tag in
+          Tag(tag: tag, site: site)
+        }
+      }
+    }
+  }
+
+  struct Header: View {
+    let context: PublishingContext<Site>
+    let selectedSection: Site.SectionID?
+
+    let sectionIDs = Site.SectionID.allCases
+
+    var body: some View {
+      VStack {
+        Link(
+          context.site.name,
+          destination: URL(string: "/", relativeTo: context.site.url)!
+        )
+        .font(.system(size: 18, weight: .bold))
+        .padding(.bottom)
+        if sectionIDs.count > 1 {
+          HStack(spacing: 16) {
+            ForEach(Array(sectionIDs), id: \.self) { section in
+              Link(destination: URL(
+                string: context.sections[section].path
+                  .absoluteString
+              )!) {
+                Text(context.sections[section].title)
+                  .underline(section == selectedSection, color: .primary)
+              }
+            }
+          }
+        }
+        // FIXME: Could not cast value of type 'Swift.Optional<TokamakCore.HStack<TokamakCore.ForEach<Swift.Array<PublishTokamak.PublishTokamak.SectionID>, PublishTokamak.PublishTokamak.SectionID, TokamakCore.ModifiedContent<TokamakCore.ModifiedContent<TokamakCore.ModifiedContent<TokamakCore.Link<TokamakCore.Text>, TokamakCore._PaddingLayout>, TokamakCore._BackgroundModifier<TokamakCore.Color>>, TokamakCore._PaddingLayout>>>>' (0x7ffac7021410) to 'TokamakCore.ViewDeferredToRenderer' (0x7ffac7021710).
```

Comment:
<!--
  1 failure:  Line should be 10...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/TokamakStaticHTML/PublishSupport/FoundationTokamakFactory+Views.swift;
  Line: 72;
-->
- :no_entry_sign: Line should be 100 characters or less: currently 505 characters (`line_length`)